src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -7174,30 +7174,42 @@ public static int Main()
     }
 }";
             CreateCompilationWithMscorlib(text).VerifyDiagnostics(
-    // (7,16): error CS1001: Identifier expected
-    //         (a, b) =>
-    Diagnostic(ErrorCode.ERR_IdentifierExpected, "=>").WithLocation(7, 16),
-    // (7,16): error CS1003: Syntax error, ',' expected
-    //         (a, b) =>
-    Diagnostic(ErrorCode.ERR_SyntaxError, "=>").WithArguments(",", "=>").WithLocation(7, 16),
-    // (7,18): error CS1002: ; expected
-    //         (a, b) =>
-    Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(7, 18),
-    // (6,9): error CS0201: Only assignment, call, increment, decrement, and new object expressions can be used as a statement
-    //         (a) => a;
-    Diagnostic(ErrorCode.ERR_IllegalStatement, "(a) => a").WithLocation(6, 9),
-    // (7,10): error CS0246: The type or namespace name 'a' could not be found (are you missing a using directive or an assembly reference?)
-    //         (a, b) =>
-    Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "a").WithArguments("a").WithLocation(7, 10),
-    // (11,9): error CS0201: Only assignment, call, increment, decrement, and new object expressions can be used as a statement
-    //         x + y; x == 1;
-    Diagnostic(ErrorCode.ERR_IllegalStatement, "x + y").WithLocation(11, 9),
-    // (11,16): error CS0201: Only assignment, call, increment, decrement, and new object expressions can be used as a statement
-    //         x + y; x == 1;
-    Diagnostic(ErrorCode.ERR_IllegalStatement, "x == 1").WithLocation(11, 16),
-    // (4,23): error CS0161: 'A.Main()': not all code paths return a value
-    //     public static int Main()
-    Diagnostic(ErrorCode.ERR_ReturnExpected, "Main").WithArguments("A.Main()").WithLocation(4, 23)
+                // (7,16): error CS1001: Identifier expected
+                //         (a, b) =>
+                Diagnostic(ErrorCode.ERR_IdentifierExpected, "=>").WithLocation(7, 16),
```

Comment:
No need to change right now. I am curious what caused changes in parser diagnostics? It doesn't look like there are parser changes.